### ISSUE TITLE
Implement acrylic 

### ISSUE THREAD

**Issue by musm** [xanderfrangos/twinkle-tray#52]
![image](https://user-images.githubusercontent.com/4319522/84950444-ecb75700-b0bc-11ea-864f-21c7257e0774.png)

![image](https://user-images.githubusercontent.com/4319522/84950458-f214a180-b0bc-11ea-9c8a-d949b464d469.png)

transparency effects are enabled, yet twinkle tray shows a solid color.

**Comment by xanderfrangos** [comment]
This is something I've looked in to many times and always hit a dead-end. There isn't a performant, bug-free way to implement Windows acrylic effects in Electron apps at this time. See this [Electron issue](https://github.com/electron/electron/issues/16391) for more info.

Once it's doable, I'll add acrylic backgrounds to Twinkle Tray. If someone has a performant, bug-free PR for this feature, I would very much welcome it.

**Comment by musm** [comment]
I see, that is indeed unfortunate!

**Comment by Seo-Rii** [comment]
What a coincidence! Recently, I developed [electron-acrylic-window](https://github.com/Seo-Rii/electron-acrylic-window), an open source that gives the vibrancy effect at Electron. I sent you the PR that applied this open source, so please check it out! https://github.com/xanderfrangos/twinkle-tray/pull/66

**Comment by Seo-Rii** [comment]
Acrylic effect is now active in Beta Branch. I think you may close the issue.

**Comment by xanderfrangos** [comment]
Acrylic blur has been added with v1.12.0. I've spent _a lot of time_ trying to get this working properly. Unfortunately, animating the brightness panel in a Windows 10-like manner has been a disaster. I've settled for a middle-ground where the Acrylic background fades in after the panel pops out. If you want a more native-like experience, there's a toggle all of the way at the bottom of the Debug menu for it. I cannot recommend it, though. Electron and Acrylic blur do not play nice when resizing (how the panel works).
